lib/dl_core/dl_core/data_source/sql.py

```diff
@@ -305,10 +305,15 @@ class TableSQLDataSourceMixin(BaseSQLDataSource):
     @property
     def table_name(self) -> str | None:
         assert isinstance(self._spec, TableSQLDataSourceSpec)
+
```

Comment:
introspective requests use table_name directly, so rendering should be here instead of sql_clause